.buildkite/run.sh

```diff
@@ -53,6 +53,10 @@ ccache -s
 # lpcxpresso11u68 boards
 pip3 install lpc_checksum
 
+# Temporary fix: Install imgtool, needed for MCUboot to sign images
+# when builindg the TF-M integration samples
+pip3 install imgtool
```

Comment:
Can't you put this in one of the pip install lines in .github/foo.yaml instead?

Reply:
In that case, I suppose it should be in `./github/workflows/sanitycheck_tests.yml`, but I am not sure if this would be better that having this temporary hack in .buildkite (which could be eventually removed, once we have a new image for CI).
Could you elaborate?

@galak any ideas (since you initially suggested this)?

Reply:
> Could you elaborate?

We have a bunch of different pip install lines throughout the github workflows which is already confusing for people and causing problems.

For example, I had to send https://github.com/zephyrproject-rtos/zephyr/pull/28922 because https://github.com/zephyrproject-rtos/zephyr/pull/28047 passed CI by editing one pip install line in one workflow used by CI, but the docs daily build used a different environment. I don't like the situation already, it's messy and people don't know where to put pip install lines anymore as seen from this example, and adding yet another place where we have to remember they might be found is not making things better.

Reply:
FIne, I will try with adding this, then, in ./github/workflows/sanitycheck_tests.yml if this is OK with you.

Reply:
> FIne, I will try with adding this, then, in ./github/workflows/sanitycheck_tests.yml if this is OK with you.

Sounds good, at least it doesn't add a new place to find pip install lines. Thanks.

Reply:
done now, @mbolivar-nordic , hope this does not break CI

Reply:
Turns out, this won't work, @mbolivar-nordic, as this workflow activates only upon changes directly on sanity check scripts. I don't think it makes sense to add a new workflow here - perhaps better to revert to the initial workaround (patching .buildkite), as there is already a similar workaround there (for `lpc_checksum`).
I can file an enhancement issue to track the removal of this workaround.

Reply:
>  I can file an enhancement issue to track the removal of this workaround.

Please do, thanks.